src/ResourceManager/Automation/Commands.Automation.Test/UnitTests/NewAzureAutomationScheduleTest.cs

```diff
@@ -320,5 +320,125 @@ public void NewAzureAutomationScheduleByHourlyWithExpiryTimeSuccessfull()
                 "Hour Frequency is unexpectedly {0}",
                 schedule.Frequency);
         }
+
+        [TestMethod]
```

Comment:
@Bpoe I'm not sure if the tests in this file are being run on CI. We may need to add the `[Fact]` and `[Trait(Category.AcceptanceType, Category.CheckIn)]` to these tests so that they are run.